GUI/src/main.tsx

```diff
@@ -51,6 +51,9 @@ const defaultQueryFn: QueryFunction | undefined = async ({ queryKey }) => {
   }
 
   const { data } = await apiInstance.get(queryKey[0] as string);
+  if(queryKey.includes('entities')) {
```

Comment:
Hey

One question about this.

if(queryKey.includes('entities')) {
    return data.response.data.entities;
  }

Cant you change it in router dsl or in the place where you calling the query?

I mean can't we leave it just to 'return data'?